taichi/program/function.h

```diff
@@ -9,6 +9,8 @@ class Program;
 
 class Function : public Callable {
  public:
+  enum class IRType { None, AST, InitialIR, OptimizedIR };
```

Comment:
nit: might need better names here. cc: @strongoier

Reply:
It seems OK for now :-)